### DISCUSSION TITLE
Sawtooth seen in Tilt and Gravity, reason?

### DISCUSSION THREAD

**Discussion by Deinjo** [universam1/iSpindel#619] [Q&A]
Hello all.

I am currently on my 2nd brew with the iSpindle. Here, too, I can see a sawtooth shape at tilt and gravity after fermentation has started (see picture). Do any of you have an idea where this could be coming from?

<img width="697" alt="2024-04-30 09_57_58-Homebrew - Dashboards - Grafana – Mozilla Firefox" src="https://github.com/universam1/iSpindel/assets/48606253/d2305c17-e09f-4a97-af90-164ec0a78a1d">

As fermentation progresses, it becomes less or almost disappears.
My first theory was that CO2 bubbles adhere to the iSpindle and thus create tilt lift.
But then bubbles would probably accumulate into large bubbles. These would then break at some point and then the angle would decrease more abruptly.

And, the sawtooth occurs more or less every 3.5-4 hours. I think that's too even and for me it points more to something technical, perhaps in the code

Have any of you observed this and have any ideas?

Thx
Deinjo

**Comment by thegreatgunbantoad** [comment]
@Deinjo looking back over my brew father brews I have never seen anything like this.  Having looked around in the code before, I don't there's anything there that would cause that.  It doesn't remember previous tilt values, it just wakes up and reports what the gryo is telling it.  There was an update to the ispindle software to improve inaccuraccy associated with the thing rolling around it's own axis, but that doesn't seem likely either.  Rather than bubbles (which would cause a sudden drop in SG (and angle) as they float off), could it be krausen buliding up and dropping off?  (The build up would lower SG (angle) which would shoot back up as it drops off).  You do seem to be logging very quickly though, presumably that's not going to a cloud service.
